js/load_jsons.js

```diff
@@ -3,7 +3,8 @@ var g_db_keys = [];
 var g_db_ref = "";
 
 $(document).ready(function() {
-  change_device("artik053");
+  var device = get_parameter_by_name('device') || 'artik053';
+  change_device(device);
```

Comment:
Maybe the parameter should be validated. 

```
supported_devices = ['stm32', 'rpi2', 'artik053'];

// Set artik053 as a default device
if (!(supported_devices.includes(device)))
  device = 'artik053';
```

Reply:
@rtakacs I don't think it is natural to redirect to `artik053` for every requests that is not in supported devices. And I belive most users don't type the url query string in url bar directly. They will just click, and it is not difficult to identify the typo or bad selection of device. But I will update as you commented. I don't want to put much time on it.

Reply:
@glistening You are correct. It is a bit strange to add artik053 as a default device if the parameter is stm32f4 (and not the required stm32).
In the next Pull Request, the default case (if no device selected, or the device parameter is not correct) will be the default web-page, so artik053 will be eliminated as a default device.

Reply:
@rtakacs Okay. I selected `artik053` as default page to keep the behaviors same to that before this PR.